apps/src/blockly/customBlocks/cdoBlockly/spritelabBlocks.js

```diff
@@ -98,4 +99,184 @@ export const blocks = {
         break;
     }
   },
+
+  installBehaviorBlocks(behaviorEditor) {
+    const generator = Blockly.getGenerator();
+    Blockly.Blocks.gamelab_behavior_get = {
```

Comment:
Starting at this line, additions in this file come from: https://github.com/code-dot-org/code-dot-org/pull/53154/files#diff-f05b95bf54e922f541e2488274f5fdf8e18f3585ecce4f95cf9555ae9c4da06aL550